devicesetup/src/main/java/io/particle/android/sdk/devicesetup/ParticleDeviceSetupLibrary.java

```diff
@@ -120,15 +172,16 @@ public static ParticleDeviceSetupLibrary getInstance() {
 
     private static ParticleDeviceSetupLibrary instance;
 
+    private SetupCompleteIntentBuilder setupCompleteIntentBuilder;
 
-    public Class<? extends Activity> getMainActivityClass() {
-        return mainActivity;
+    public SetupCompleteIntentBuilder getSetupCompleteIntentBuilder() {
```

Comment:
couldn't this just be package visible? I'd like to keep the public API surface as small as possible.